src/app.py

```diff
@@ -72,37 +77,71 @@ def test_endpoint():
     },
 ]
 
-
-#@app.route("/fortune", methods=["GET"])
-
+# Helper function
 def get_all_fortunes():
     # connect to the database
     dynamodb = boto3.resource('dynamodb', region_name='us-west-2')
 
     # Access database table
     table = dynamodb.Table('FortuneCookie')
 
-    fortune = "Your high-minded principles spell success."
-
-    #response_obj = {"status": "success"}
     try:
         response_obj = table.scan()
     except ClientError as e:
-        print("Failed...")
         print(e.response['Error']['Message'])
     else:
         items = response_obj['Items']
         return items
         # testing prints
         #print("GetItem succeeded: ")
         #print(json.dumps(item, indent=4, cls=DecimalEncoder))
-    #return jsonify(response_obj) this is for Dustin
 
+def delete_fortune(id):
+    # connect to the database
+    dynamodb = boto3.resource('dynamodb', region_name='us-west-2')
 
+    # Access database table
+    table = dynamodb.Table('FortuneCookie')
+
+    try:
+        response = table.delete_item(
+            Key={
+                'id' : id
+            }
+        )
+    except ClientError as e:
+        if e.response['Error']['Code'] == "ConditionalCheckFailedException":
+            print(e.response['Error']['Message'])
+        else:
+            raise
+    else:
+        print("DeleteItem succeeded: ")
+        print(json.dumps(response, indent=4))
+    
+@app.route("/fortune", methods=["GET"])
+def get_fortune():
+    fortunes = get_all_fortunes()
+
+    # Choose a random fortune
+    chosen = random.randint(0, len(fortunes)-1)
```

Comment:
2 things. 1) use `random.choice(list))` so the code is more readable which allows 2) removing of the comment

Reply:
Updated. :)